src/codeu/chat/client/core/Context.java

```diff
@@ -47,4 +49,12 @@ public UserContext create(String name) {
     }
     return users;
   }
+
+  public ServerInfo getVersion() {
+    return view.getVersion();
+
+  public ServerInfo getServerUptime() {
+    return view.getServerUptime();
+
```

Comment:
Nit: Please remove the extra blank line here. (@lysanderp41 , or maybe @mlk209 if this came up during the merge)